app/models/kitchen.rb

```diff
@@ -1,3 +1,5 @@
 class Kitchen < ApplicationRecord
   has_many :appliances
+
+  validates :floor, uniqueness: true
```

Comment:
The uniqueness validation is not transaction safe, so we use it with database foreign keys in practice. 
More about this in the[ validation guide](https://guides.rubyonrails.org/active_record_validations.html#uniqueness).

We should talk about this in a separate PR. I'll create an issue for it.